frontend/packages/core/src/index.tsx

```diff
@@ -1,6 +1,8 @@
 import { Grid } from "@material-ui/core";
 
 import { userId } from "./AppLayout/user";
+import LinearTimeline from "./Charts/linearTimeline";
```

Comment:
You should be able to do:

```
export { default as LinearTimeline } from "./Charts/linearTimeline";
```

Although it might be better to create a Charts index file and just export everything from there instead.